### PR TITLE
[F] Add a thing to put the ontology information into a graph database

### PR DESCRIPTION
**Pull request by petereast**
- [x] Initialise library
- [x] Set up graph db
- [x] Read ontology file
- [x] Insert ontologies into Arangodb
- [x] Create collection of relationships
- [x] Create a graph from that collection

### CODE REVIEW

**Comment by petereast** [comment]
For another PR: A thing to run this with docker-compose up

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  redis:
+    image: redis:5
+    ports:
+      - "6379:6379"
+  arango:
+    image: arangodb:3.2
+    ports:
+      - "8529:8529"
+    environment:
+      - ARANGO_NO_AUTH=1
```
Comment (mareq) [inline docker-compose.yaml:12]: :rhinoceros: :old_key: :closed_lock_with_key:

Comment (petereast) [reply docker-compose.yaml:12]: Obviously not a production setup :see_no_evil:

**Review by mareq — APPROVED** [review]
:koala: :bamboo: :australia:

**Pull request merged**